test/iterative_verifier_test.go

```diff
@@ -82,6 +105,28 @@ func (t *IterativeVerifierTestSuite) TestVerifyOncePass() {
 	t.Require().Equal("", result.Message)
 }
 
+func (t *IterativeVerifierTestSuite) TestVerifyCompressedOncePass() {
+	t.InsertCompressedRowInDb(42, testhelpers.TestCompressedData1, t.Ferry.SourceDB)
+	t.InsertCompressedRowInDb(42, testhelpers.TestCompressedData1, t.Ferry.TargetDB)
+
+	result, err := t.verifier.VerifyOnce()
+	t.Require().NotNil(result)
+	t.Require().Nil(err)
+	t.Require().True(result.DataCorrect)
+	t.Require().Equal("", result.Message)
+}
+
+func (t *IterativeVerifierTestSuite) TestVerifyCompressedMismatchOncePass() {
+	t.InsertCompressedRowInDb(43, testhelpers.TestCompressedData3, t.Ferry.SourceDB)
```

Comment:
Let's assert our test's assumption right before this with a helpful comment (after all it's the whole point of the PR):

```
         // Two fixtures that have different compressed values but have equal decompressed values
	t.Require().NotEqual(testhelpers.TestCompressedData3, testhelpers.TestCompressedData4)
```